common/rnexecutorch/data_processing/Numerical.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Numerical.h"
+
+#include <algorithm>
+#include <numeric>
+
+namespace rnexecutorch::numerical {
+void softmax(std::vector<float> &v) {
+  float max = *std::max_element(v.begin(), v.end());
+
+  for (float &x : v) {
+    x = std::exp(x - max);
+  }
+  float sum = std::accumulate(v.begin(), v.end(), 0.f);
+  for (float &x : v) {
+    x /= sum;
+  }
+}
```

Comment:
Looks familiar, move softmax from previous PR to utilities if haven't done so, and reuse it here

Reply:
Ok, I see that this is the same location as in image segmentation. Please rebase then